src/components/containers/visibility.tsx

```diff
@@ -37,11 +38,14 @@ const domainBasedCheck = () => {
 
     useEffect(() => {
         if (window) {
-            const host_name = window.location.hostname
-            if (host_name.includes('eu')) {
+            const subdomain = window.location.hostname.split('.').slice(0, -2).join('.')
+            const eu_domains = ['eu', 'staging-eu']
+            const uk_domains = ['uk', 'staging-uk']
```

Comment:
If there is a config file in the project to store constants like these, it's better to move these to the config file.